.eslintrc.js

```diff
@@ -89,6 +89,7 @@ module.exports = {
     '@typescript-eslint/typedef': 'off',
     '@typescript-eslint/unbound-method': 'error',
     '@typescript-eslint/unified-signatures': 'error',
+    '@typescript-eslint/no-non-null-assertion': 0,
```

Comment:
this will disable `no-non-null-assertion` for all our codebase, but we need to disable it only in `test/source` folder

Reply:
Yep!. we don't need to disable `no-non-null-assertion` for all our codebase.
Just pushed fix